asv_bench/benchmarks/benchmarks.py

```diff
@@ -820,3 +834,212 @@ def time_index(self, shape):
 
 
 from .utils import setup  # noqa: E402, F401
+
+
+class NumericSeriesIndexing:
```

Comment:
Just to be consistent in class names. 

```suggestion
class TimeIndexingSeries:
```

Could you make this class more like `TimeIndexing`? To do this, you need to add a new parameter: `shape` and use `get_benchmark_shapes` that will define values for the parameter.

Reply:
However, at the same time, you should try to keep the specifics of this benchmark.

Reply:
Have Renamed classes and added shape parameter for benchmark classes